src/transaction/guards.ts

```diff
@@ -25,6 +25,14 @@ export const isUint64 = (value: any): value is number => {
   } catch (e) { return false }
 }
 
+export const isInt64 = (value: any): value is number => {
+  if (typeof value === 'number') return true
+  try {
+    BigInt(value)
```

Comment:
This does not enforce any upper limit on the value, so it might not fit in 8 bytes?